src/kernel/lthread/lthread_sched_wait.h

```diff
@@ -25,9 +25,9 @@ struct sys_timer;
  */
 struct sched_wait_info {
 	struct sys_timer *tmr;
-	clock_t remain;
-	clock_t last_sync;
-	int status;
+	clock_t			  remain;
```

Comment:
Tab used to align. Tabs should be used only to indentation, alignment should be only with spaces, that will make good look with every tab length in most cases